src/tarkov-data-manager/modules/remote-data.mjs

```diff
@@ -208,25 +224,31 @@ const methods = {
                     continue;
                 }
 
-                const lastLowData = lastLowPriceResults.find(row => row.item_id === itemId);
-                if (lastLowData) {
-                    item.lastLowPrice = lastLowData.price;
-                    item.updated = lastLowData.timestamp;
-                }
-
-                item24hPrices[itemId]?.sort();
-                item.avg24hPrice = getInterquartileMean(item24hPrices[itemId] || []) || null;
-                item.low24hPrice = item24hPrices[itemId]?.at(0);
-                item.high24hPrice = item24hPrices[itemId]?.at(item24hPrices[itemId]?.length - 1);
+                for (let pve = 0; pve < 2; pve++) {
+                    const fieldPve = pve ? pveSuffix : '';
 
-                const itemPriceYesterday = avgPriceYesterday.find(row => row.item_id === itemId);
-                if (!itemPriceYesterday || item.avg24hPrice === 0) {
-                    item.changeLast48h = 0;
-                    item.changeLast48hPercent = 0;
-                } else {
-                    item.changeLast48h = Math.round(item.avg24hPrice - itemPriceYesterday.priceYesterday);
-                    const percentOfDayBefore = item.avg24hPrice / itemPriceYesterday.priceYesterday;
-                    item.changeLast48hPercent = Math.round((percentOfDayBefore - 1) * 100 * 100) / 100;
+                    const lastLowData = lastLowPriceResults.find(row => row.item_id === itemId && row.pve === pve);
+                    if (lastLowData) {
+                        item[`lastLowPrice${fieldPve}`] = lastLowData.price;
+                        if (!pve) {
+                            item.updated = lastLowData.timestamp;
+                        }
+                    }
+    
+                    item24hPrices[pve][itemId]?.sort();
+                    item[`avg24hPrice${fieldPve}`] = getInterquartileMean(item24hPrices[pve][itemId] || []) || null;
+                    item[`low24hPrice${fieldPve}`] = item24hPrices[pve][itemId]?.at(0);
+                    item[`high24hPrice${fieldPve}`] = item24hPrices[pve][itemId]?.at(item24hPrices[pve][itemId]?.length - 1);
+    
+                    const itemPriceYesterday = avgPriceYesterday.find(row => row.item_id === itemId && row.pve === pve);
+                    if (!itemPriceYesterday || item[`avg24hPrice${fieldPve}`] === 0) {
+                        item[`changeLast48h${fieldPve}`] = 0;
+                        item[`changeLast48hPercent${fieldPve}`] = 0;
+                    } else {
+                        item[`changeLast48h${fieldPve}`] = Math.round(item[`avg24hPrice${fieldPve}`] - itemPriceYesterday.priceYesterday);
+                        const percentOfDayBefore = item[`avg24hPrice${fieldPve}`] / itemPriceYesterday.priceYesterday;
+                        item[`changeLast48hPercent${fieldPve}`] = Math.round((percentOfDayBefore - 1) * 100 * 100) / 100;
```

Comment:
I think it's better to have a separate endpoint for pvp/pve prices. This way a simple switch of endpoint makes everything else work flawlessly.
While with this approach we need to rewrite and duplicate all sort of code and calculations... 🤔

Reply:
These are just the fields for storing the price data. I'm thinking that on the API side of things, using a PVE arg will cause these prices to be used instead of the other prices.